src/math/fmod.c

```diff
@@ -12,11 +12,11 @@
 
 double
 fmod(double x, const double y) {
-  if (x > y && x > 0 && y > 0) {
-    while (x >= y) {
-      x -= y;
-    }
-    return x;
+  if (x < y || x < 0 || y < 0) {
+    return 0;
```

Comment:
Note that this is not exactly the inverse of the original logical condition, as the inverse of `>` is not `<` but rather `<=`.